src/bin/fullnode.rs

```diff
@@ -181,7 +181,7 @@ fn main() {
         loop {
             let last_id = client.get_last_id();
             let transaction =
-                VoteTransaction::vote_account_new(&keypair, vote_account_id, last_id, 1, 1);
+                VoteTransaction::vote_account_new(&keypair, vote_account_id, last_id, 1, 0);
```

Comment:
Instead of ignoring the fee, which is unrealistic, let's increase the amount of tokens in the vote account when it gets created [1] or airdropped [2] please

[1] https://github.com/solana-labs/solana/blob/d8fce2cbd483653f7e3d0814e7e39834898cda9a/src/bin/genesis.rs#L21
[2] https://github.com/solana-labs/solana/blob/d8fce2cbd483653f7e3d0814e7e39834898cda9a/multinode-demo/fullnode.sh#L167

Reply:
Updated the airdrop tokens to 3